suites/reef/nfs/tier1-nfs-ganesha-upgrade-ibm-7x-to-7x-latest.yaml

```diff
@@ -0,0 +1,212 @@
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              command: apply
+              service: mds
+              base_cmd_args:
+                verbose: true
+              pos_args:
+                - cephfs
+              args:
+                placement:
+                  nodes:
+                    - node2
+                    - node6
+                  limit: 2
+                  sep: " "
+          - config:
+              args:
+                - "ceph osd pool create rbd"
+              command: shell
+          - config:
+              args:
+                - "rbd pool init rbd"
+              command: shell
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Configure client1
+      desc: Configure the RGW,RBD client system
+      module: test_client.py
+      polarion-id: CEPH-83573777
+      config:
+        command: add
+        id: client.1
+        node: node4
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Configure client2
+      desc: Configure the RGW,RBD client system
+      module: test_client.py
+      polarion-id: CEPH-83573777
+      config:
+        command: add
+        id: client.2
+        node: node5
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Configure client3
+      desc: Configure the RGW,RBD client system
+      module: test_client.py
+      polarion-id: CEPH-83573777
+      config:
+        command: add
+        id: client.3
+        node: node6
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Configure client4
+      desc: Configure the RGW,RBD client system
+      module: test_client.py
+      polarion-id: CEPH-83573777
+      config:
+        command: add
+        id: client.4
+        node: node7
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Upgrade cluster to latest IBM 7.x along with NFS-Ganesha IOs
+      desc: Upgrade cluster to latest IBM 7.x along with NFS-Ganesha IOs
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: Nfs Verify multiple parallel io and lookups
+            module: nfs_verify_multiple_parallel_io_and_lookups.py
+            desc: Perform look ups while multiple parallel io are in progress
+            polarion-id: CEPH-83581304
+            abort-on-fail: false
+            config:
+              nfs_version: 4.1
+              clients: 4
```

Comment:
Each Test is creating NFS cluster and deleting the cluster post performing the operations. Better to have all operations on 1 single NFS cluster without teardown and later upgrade the same NFS cluster

Reply:
Hi @msainiRedhat 

As of now we not have any test for just creating cluster without teardown 

As per the discussion with "Pranav" as of now we are putting existing test case, Later once we received the 7.1 build we will create test specific to upgrade functionlity

Reply:
Noted